nominees/opensourceecology.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "OpenSourceEcology",
```

Comment:
@Msgl could you update the name field to "Open Source Ecology" as represented on the website. Thanks.